### PR TITLE
Create new ubuntu_unused_kernels gem

### PR DESCRIPTION
**Pull request by dcarley**
This is a rewrite of our internal utility [`govuk_unused_kernels`](https://github.gds/gds/puppet/blob/master/modules/govuk/files/bin/govuk_unused_kernels). I've done this because:
- I'd like to run this on cron so that we don't have to do it manually but I was nervous about doing so because it didn't have any tests
- it's good to open source software

If merged then I will:
- [ ] Move the repo to the gds-operations organisation
- [ ] Setup a job on `ci-new` so that it publishes to rubygems.org
- [ ] Remove/replace our internal copy of this in gds/puppet

### CODE REVIEW

**Comment by samjsharpe** [comment]
Needs a LICENCE before you can publish it ;)

**Comment by dcarley** [comment]
> Needs a LICENCE before you can publish it ;)

There's a `LICENSE.txt` which comes from the master branch that this is merging against.

**Comment by samjsharpe** [comment]
Ignore me then. I saw the initial commit in the list and assumed it was an initial commit :sweat_smile:

**Comment by mattbostock** [comment]
:eggplant:

File: ubuntu_unused_kernels.gemspec [hunk 1]
```diff
@@ -8,9 +8,9 @@ Gem::Specification.new do |spec|
   spec.version       = UbuntuUnusedKernels::VERSION
   spec.authors       = ["Dan Carley"]
   spec.email         = ["dan.carley@gmail.com"]
```
Comment (ajlanghorn) [inline ubuntu_unused_kernels.gemspec:10]: The Gemspec for other projects lists a @digital.cabinet-office.gov.uk mailing list as the spec.email value; specifically vCloud Tools and a few others. Do we want to do the same here?

**Pull request merged**